components/automate-cli/cmd/chef-automate/version.go

```diff
@@ -3,16 +3,27 @@
 package main
 
 import (
+	"container/list"
 	"context"
+	"errors"
+	"fmt"
 	"os"
+	"regexp"
+	"strings"
 
+	"github.com/pelletier/go-toml"
+	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 
 	api "github.com/chef/automate/api/interservice/deployment"
 	"github.com/chef/automate/components/automate-cli/pkg/docs"
 	"github.com/chef/automate/components/automate-cli/pkg/status"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/services/statusservice"
 	"github.com/chef/automate/components/automate-deployment/pkg/client"
+	"github.com/chef/automate/lib/stringutils"
 	"github.com/chef/automate/lib/version"
+	"github.com/ttacon/chalk"
```

Comment:
please use `github.com/fatih/color` to render the styled text. We are using this package in our repository. You can avoid importing the new package